src/wallet/rpcwallet.cpp

```diff
@@ -3184,7 +3232,8 @@ static RPCHelpMan fundrawtransaction()
                 "Note that inputs which were signed may need to be resigned after completion since in/outputs have been added.\n"
                 "The inputs added will not be signed, use signrawtransactionwithkey\n"
                 " or signrawtransactionwithwallet for that.\n"
-                "Note that all existing inputs must have their previous output transaction be in the wallet.\n"
+                "Note that all existing inputs must either have their previous output transaction be in the wallet\n"
+                "or be in the utxo set. Solving data must be provided for non-wallet inputs.\n"
```

Comment:
Maybe we should require the user to provide the TXO too?

Reply:
I don't think it should be required. It could be an option like for the `signrawtransaction*` RPCs.